client/state/data-layer/wpcom/activity-log/rewind/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import {
+	rewindStatusError,
+	updateRewindStatus,
+} from 'state/activity-log/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchRewindStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
+		apiVersion: '1',
+	}, action ) );
+};
+
+const fromApi = response => ( {
+	active: response.use_rewind,
+	firstBackupDate: response.first_backup_when,
+	isPressable: response.is_pressable,
```

Comment:
we might need to make sure this actually comes across as a boolean. often I see `0` or `1` or `"true"` or goofy values like that